src/Helper/prettier/index.ts

```diff
@@ -0,0 +1,32 @@
+import path from "path";
+import fs from "fs";
+import { format } from "prettier";
```

Comment:
Prettier is left in the `devDependencies`

Reply:
I moved it 🚀 package-lock.json is collapsed by default, you can see the change there.